packages/@okta/vuepress-site/docs/guides/authenticators-google-authenticator/main/ios/enrollmentintegrationsteps.md

```diff
@@ -0,0 +1,550 @@
+>**Note:** The examples in this guide use Swift 5, SwiftUI, and a target of iOS 15 or later.
+
+The examples in this guide manage the sign-on flow using a singleton class (`SignInController`). The class handles asynchronous responses from the SDK by implementing the delegate functions of the `IDXClientDelegate` protocol. The SDK also provides completion handlers and Swift Concurrency (`async/await`).
+
+The SwiftUI portions pass state information using multiple bindings, consider using more robust ways of sharing and updating data in production apps. For more information, see the [Apple developer documentation for state and data flow in SwiftUI](https://developer.apple.com/documentation/swiftui/state-and-data-flow/).
```

Comment:
cross reference/link wording is the use of just "See" and not "for more information" unless it's unclear what information the link is taking the reader to - then provide "for more information on blah blah blah"

```suggestion
The SwiftUI portions pass state information using multiple bindings, consider using more robust ways of sharing and updating data in production apps. See the [Apple developer documentation for state and data flow in SwiftUI](https://developer.apple.com/documentation/swiftui/state-and-data-flow/).
```

Reply:
It's my Apple DevPubs finger memory where "For more information, see" was the standard. Fixed and in the next commit.